ai/writing-assistant.rst

```diff
@@ -0,0 +1,83 @@
+Writing Assistant
+=================
+
+Zammad's writing assistant tools allow agents to apply AI based changes to
+their text before saving or sending it. You can access the settings in Zammad's
+admin section under *AI > Writing Assistant*. Make sure to configure an
+:doc:`AI provider <provider>` before activating the feature. Otherwise, a
+warning message will tell you to do so.
+
+Default Tools
+-------------
+
+Zammad ships some default tools:
+
+- Improve writing
+- Fix spelling and grammar
+- Expand
+- Simplify
+
+These are useful basic tools for all agents. This is why they are not
+customizable and deletable. If you enable the feature by turning the
+**Writing Assistant** toggle on, these default tools are available to all
+agents.
+
+Note that these built-in tools can benefit from optimizations which can be
+shipped by an update of Zammad.
```

Comment:
Nope: similar to other DB seeds, we will *not* migrate existing text tools, since they could be modified in unforeseen ways. Because of that, only new instances will benefit from prompt updates for existing tools, unless we decide to add new ones in the future.

I suggest we remove this part.